---
title: "DNA methylation and hydroxymethylation"
output: html_document
---

## reviews
* [Wu, X. and Y. Zhang (2017). "TET-mediated active DNA demethylation: mechanism, function and beyond." Nat Rev Genet 18(9): 517-534.](https://www.nature.com/articles/nrg.2017.33)
* [Wu, H. and Y. Zhang (2015). "Charting oxidized methylcytosines at base resolution." Nat Struct Mol Biol 22(9): 656-661.](https://www.nature.com/articles/nsmb.3071)
* [Wu, H. and Y. Zhang (2014). "Reversing DNA methylation: mechanisms, genomics, and biological functions." Cell 156(1-2): 45-68.](http://www.cell.com/cell/fulltext/S0092-8674(13)01588-2)
* [Smith, Z. D. and A. Meissner (2013). "DNA methylation: roles in mammalian development." Nat Rev Genet 14(3): 204-220](https://www.nature.com/articles/nrg3354)  
* [Jones, P. A. (2012). "Functions of DNA methylation: islands, start sites, gene bodies and beyond." Nat Rev Genet 13(7): 484-492](https://www.nature.com/articles/nrg3230)   


## 5hmC genome-wide studies at single-base level
* [Wen, L., et al. (2014). "Whole-genome analysis of 5-hydroxymethylcytosine and 5-methylcytosine at base resolution in the human brain." Genome Biol 15(3): R49.](https://genomebiology.biomedcentral.com/articles/10.1186/gb-2014-15-3-r49)
* [Yu, M., et al. (2012). "Base-resolution analysis of 5-hydroxymethylcytosine in the mammalian genome." Cell 149(6): 1368-1380.](https://www.sciencedirect.com/science/article/pii/S009286741200534X)  
    TAB-seq method
* [Booth, M. J., et al. (2012). "Quantitative sequencing of 5-methylcytosine and 5-hydroxymethylcytosine at single-base resolution." Science 336(6083): 934-937.](http://science.sciencemag.org/content/336/6083/934.full)  
    oxBS-seq method
    

