---
title: "Single Cell Sequencing"
output: html_document
---
## reviews
* [Kelsey, G., et al. (2017). "Single-cell epigenomics: Recording the past and predicting the future." Science 358(6359): 69-75.](http://science.sciencemag.org/content/358/6359/69)<br>
* [Macaulay, I. C., et al. (2017). "Single-Cell Multiomics: Multiple Measurements from Single Cells." Trends Genet 33(2): 155-168.](http://www.sciencedirect.com/science/article/pii/S016895251630169X?via%3Dihub)<br>
* [Baslan, T. and J. Hicks (2017). "Unravelling biology and shifting paradigms in cancer with single-cell sequencing." Nat Rev Cancer 17(9): 557-569.](https://www.nature.com/articles/nrc.2017.58)<br>

## 方法学reviews
* [Chen, J., et al. (2018). "Constructing cell lineages from single-cell transcriptomes." Mol Aspects Med 59: 95-113.](https://www.sciencedirect.com/science/article/pii/S0098299717300523?via%3Dihub)

## 工具软件比较
* [Freytag, S., et al. (2018). "Comparison of clustering tools in R for medium-sized 10x Genomics single-cell RNA-sequencing data." F1000Res 7: 1297.](https://f1000research.com/articles/7-1297/v2)<br>
    benchmark各个软件的clustering效果  
* [Saelens, W., et al. (2019). "A comparison of single-cell trajectory inference methods." Nat Biotechnol 37(5): 547-554.](https://www.nature.com/articles/s41587-019-0071-9)  
    对目前所有TI软件进行评价，并封装到一个软件之下，方便用户运行多种TI方法   


## single cell相关学习资料
* [github awesome-single-cell](https://github.com/seandavi/awesome-single-cell)
* [scRNA-seq分析教程](https://hemberg-lab.github.io/scRNA.seq.course/)
* [f1000research上的scRNA-seq workflow](https://f1000research.com/articles/5-2122/v1)